plugins/azure_sentinel/help.md

```diff
@@ -373,7 +1650,7 @@ Example input:
 ```
 {
   "incidentId": "incident123",
-  "resourceGroupName": "resourcegrup1",
+  "resourceGroupName": "resourcegroup1",
   "subscriptionId": "0caafeeb-aaa0-44ca-ffe1-aaaaeeeffffe",
```

Comment:
I think this may be a real value 😬 Please search for everywhere we see `"subscriptionId": "0caafeeb-aaa0-44ca-ffe1-aaaaeeeffffe"` and replace it with a randomly generated UUID.